### PR TITLE
feat(mnq): add regionality, subscriptions & no-FIFO

### PR DESCRIPTION
**Pull request by RoRoJ**
- Regionality
- SNS subscriptions
- No-more SNS FIFO
- Added cross-links to new tutos, updated product page
- Remove old namespace pages definitively

### CODE REVIEW

File: serverless/messaging/concepts.mdx [hunk 1]
```diff
@@ -139,13 +150,21 @@ Stream persistence is one of the factors affecting the billing of Scaleway Messa
 
 In [publish/subscribe](#publishsubscribe) systems such as [SNS](#sns), a subscriber is the entity (e.g. SQS queue, function, URL) that messages from topics are pushed to. Subscribers can filter messages based on their topic or content.
```
Comment (nerda-codes) [inline serverless/messaging/concepts.mdx:151]: Not sure about this one, but shouldn't there be a slash between "publish" and "subscribe" in the anchor?

File: serverless/messaging/how-to/create-manage-subscriptions.mdx [hunk 2]
```diff
@@ -0,0 +1,118 @@
...
+  paragraph: This page explains how to create and manage SNS subscriptions
+tags: messaging queuing subscriptions sns topics
+categories:
+  - serverless 
+dates:
+  validation: 2024-02-06
+  posted: 2024-02-06
+---
+
+This page shows how to create and manage SNS [subscriptions](/serverless/messaging/concepts/#topic) with Scaleway Messaging and Queuing. When you create a subscription to a [topic](/serverless/messaging/concepts/#topic), you are effectively signing up an endpoint to receive all messages published to that topic.
```
Comment (nerda-codes) [inline serverless/messaging/how-to/create-manage-subscriptions.mdx:16]:
```suggestion
This page shows how to create and manage SNS [subscriptions](/serverless/messaging/concepts/#subscription) with Scaleway Messaging and Queuing. When you create a subscription to a [topic](/serverless/messaging/concepts/#topic), you are effectively signing up an endpoint to receive all messages published to that topic.
```

File: serverless/messaging/how-to/create-manage-subscriptions.mdx [hunk 3]
```diff
@@ -0,0 +1,118 @@
...
+
+    The subscription is created. If you created an HTTP or HTTPS subscription, you will now need to **confirm** it, as described below. This step is not required for Serverless Functions or Containers subscriptions.
+
+## How to confirm an HTTP or HTTPS subscription
+
+An HTTP or HTTPS subscription will have a `Pending` status until you **confirm** it. This is essential for it to be able to start receiving messages. Follow the steps below to confirm the subscription:
+
+1. Access the subscription's **Settings** page (from the console side menu, **Messaging** > **SNS** > **Topics**, click the topic containing the subscription, then click the subscription).
+
+2. Click the **Confirm Subscription** button at the top right of the page. A popup displays:
```
Comment (nerda-codes) [inline serverless/messaging/how-to/create-manage-subscriptions.mdx:81]:
```suggestion
2. Click the **Confirm Subscription** button at the top right of the page. A pop-up displays:
```

File: serverless/messaging/concepts.mdx (hunk 1 above)
Comment (RoRoJ) [reply serverless/messaging/concepts.mdx:151]: No, I checked

**Review by bene2k1 — APPROVED** [review]

**Pull request merged**